src/test/rustdoc-gui/search-form-elements.goml

```diff
@@ -222,7 +222,7 @@ assert-css: (
 assert-css: (
     "#settings-menu > a",
     {
-        "color": "rgb(56, 115, 173)",
+        "color": "rgb(0, 0, 0)",
```

Comment:
So about this: it was supposed to test the color of the links inside the help. Can you fix the path and revert the color check please?

Reply:
Oh right sorry, thought it was the help menu, didn't pay attention to the CSS selector, sorry.